composer.json

```diff
@@ -21,7 +21,7 @@
         "composer/package-versions-deprecated": "1.11.99.2",
         "csa/guzzle-bundle": "^2.2",
         "cweagans/composer-patches": "^1.6",
-        "elife/api": "^2.9",
+        "elife/api": "dev-develop",
```

Comment:
To introduce as a `--dev` dependency:

`composer require --dev elife/api:^2.9`